### PR TITLE
EN-1985-Support-RadioGroup-in-PAM-forms-widget

### PR DESCRIPTION
**Pull request by alepintus**
Hi @joewhite101 ,
RadioGroup added.
Could you please have a look?
Thanks!

Alessio

### CODE REVIEW

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieVersionTransformer.java [hunk 1]
```diff
@@ -381,6 +384,26 @@ private static KieProcessFormField buildField(  String id, String modelName,
 
         }
         
+        if (field.getType().equals("RadioGroup")) {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieVersionTransformer.java]: Not specific to this PR but we need to add logging for the case where we encounter a fieldType we don't recognize. Should be a WARN

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alepintus** [comment]
retest this please

**Pull request merged**